members-service/request-schemas.js

```diff
@@ -18,14 +18,19 @@ export const SHOW_MEMBER_SCHEMA = {
 
 export const UPDATE_MEMBER_SCHEMA = {
   type: 'object',
-  required: ['id', 'email'],
+  required: ['id', 'email', 'member'],
   properties: {
-    id: { type: 'string', pattern: '^[0-9]+$' },
     email: { type: 'string', format: 'email' },
-    first_name: { type: 'string', minLength: 2 },
-    last_name: { type: 'string', minLength: 2 },
-    country: { type: 'string', minLength: 2 },
-    postal: { type: 'string', minLength: 2 },
+    id: { type: 'string', pattern: '^[0-9]+$' },
+    member: {
+      type: 'object',
+      properties: {
+        first_name: { type: 'string', minLength: 2 },
```

Comment:
You didn't add email to the member properties to be passed.

Reply:
great catch! Although I'm not sure how this was working given this code...